src/NuGet.Services.AzureSearch/Auxiliary2AzureSearch/Auxiliary2AzureSearchConfiguration.cs

```diff
@@ -12,6 +12,7 @@ public class Auxiliary2AzureSearchConfiguration : AzureSearchJobConfiguration, I
         public string AuxiliaryDataStorageContainer { get; set; }
         public string AuxiliaryDataStorageDownloadsPath { get; set; }
         public string AuxiliaryDataStorageVerifiedPackagesPath { get; set; }
+        public string AuxiliaryDataStorageExcludedPackagesListPath { get; }
```

Comment:
Having "List" in the name is too verbose. For example, VerifiedPackages is not VerifiedPackagesList. I think in the blob name and in the property names it should be "ExcludedPackages"